posorneg.c

```diff
@@ -0,0 +1,22 @@
+/* Christopher Liu - gets a number, checks to see if it's greater or less than zero. If neither, assumes its zero. */
```

Comment:
Nice job! The only thing I'd add is a message in the beginning that prompts the user. Besides that, nice job!